src/app/frontend/secret/detail/info.html

```diff
@@ -19,5 +19,10 @@
   <kd-info-card-section>
     <kd-object-meta-info-card object-meta="::$ctrl.secret.objectMeta">
     </kd-object-meta-info-card>
+    <kd-info-card-entry title="[[Type|Secret info details section type entry.]]">
+      <kd-middle-ellipsis display-string="{{::$ctrl.secret.type}}">
+      </kd-middle-ellipsis>
+      <div ng-hide="::$ctrl.secret.type">-</div>
```

Comment:
This does not match https://github.com/kubernetes/dashboard/pull/2256/files#diff-a2f3442e2ae2857ec87f5b2601a01f96R1187.

```